app/src/main/res/layout/fragment_history.xml

```diff
@@ -116,6 +116,8 @@
                 android:clipToPadding="false"
                 android:orientation="vertical"
                 android:paddingTop="24dp"
+                android:paddingBottom="@dimen/history_transaction_list_last_item_padding"
+                app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
```

Comment:
I would add layout manager in code, it would be more obvious for the developer to understand how is recycler looked like

Reply:
I would rather add it here, 
it's always a boilerplate that even 3rd party libraries hide inside their own RecyclerViews. If you can declare smth in xml you should do it and remove these lines from the Fragment that has a lot already

Reply:
You see all params for recycler including layout manager in Fragment and remove all lines again if necessary as you said.
From the other side, setting layout manager in xml is native sdk solution and that's ok to use it.
In general, it's not a critical question, I gave you a case to think about. Please leave it if you don't think to move it is neccesary

Reply:
What do our colleagues have to say? @kabramovp2p @kuznetsov @davran1001

Reply:
i do not like setting LinearLayoutManager in xml
so i am against of this =)

Reply:
it depends on situation i usually prefer to use xml for this